### DISCUSSION TITLE
Parallel Plugin Calls

### DISCUSSION THREAD

**Discussion by wupengfei0904** [danny-avila/LibreChat#1990] [Feature Requests & Suggestions]
### What features would you like to see added?

It is hoped that the flexibility of GPT4 to call plug-ins can be increased. 3 plug-ins are currently selected, but GPT4 is unable to call these plug-ins in a coordinated manner, and only one plug-in is executed at a time, and it needs to be re-entered in order to execute the next step.

### More details

Implement chatgpt's plugin invocation functionality so that when a user selects a plugin, it is up to gpt4 to decide when and how to use a plugin, and the result of one plugin is the input to another. The final output is summarized by gpt4, rather than ending the round of conversation after executing a plugin.

### Which components are impacted by your request?

Plugins

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
For the closest experience to this now, use the Assistants API. Parallel function calling and everything else you said is done through this functionality.

In the future, the current Plugins experience will be removed in favor of adding tools to any endpoint.
